internals/scripts/CheckNativeDep.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import fs from 'fs';
+import chalk from 'chalk';
+
+const EndOfLine = require('os').EOL;
+const { execSync } = require('child_process');
+
+const locateNativeDeps = async () => {
```

Comment:
How about this?
```js
const locateNativeDeps = async () => {
  return fs.readdirSync('node_modules')
    .filter(folder => fs.existsSync(`node_modules/${folder}/binding.gyp`))
};
```

Reply:
Agreed, but got any idea on how we'd ignore node-sass or remove it later from the array?

Reply:
Nvm, got it